js/test/Exchange/test.fetchFundingRateHistory.js

```diff
@@ -6,12 +6,12 @@ const assert = require ('assert')
 
 // ----------------------------------------------------------------------------
 
-module.exports = async (exchange) => {
+module.exports = async (exchange, symbol) => {
 
     const method = 'fetchFundingRateHistory'
 
     const format = {
-        'currency': 'USDT',
+        'symbol': 'BTC/USDT',
```

Comment:
Should we be using a spot symbol here?

Reply:
those values in tests' `format` object doesn't actually matter. it only matter that they had the correct keyname (and value type). the actual values doesnt matter and is nowhere used, you can even input there `'symbol': 'blabla'`.  however, i've inserted just the standard string for it.

Reply:
you can easily confirm that if you look into other sibling test files.

Reply:
but on the other hand, it doesn't hurt, let's add the `:USDT` in string value.